client/src/components/ListEntry.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import StarRating from './StarRating';
 
 export default class ListEntry extends React.Component {
   constructor(props) {
```

Comment:
note for below:

Setting state based on props is an antipattern. This creates two sources of truth. If you need to read values from props, just read them from props.

Reply:
Fixed it!